api/commands/misc/timezone.js

```diff
@@ -52,11 +52,10 @@ module.exports = class TimezoneCommand extends Command {
         reject(msg.say(exceptions.invalid_timezone)));
     }
 
-    return insertTz([
-      msg.message.author.username,
-      msg.message.author.discriminator,
-      tz
-    ])
-    .then(() => msg.direct(`Your timezone has been set to ${tz}`));
+    return tzStore.setAsync(msg.message.author.id, tz)
+    .then(() => msg.direct(`Your timezone has been set to ${tz}`))
+    .catch((err) => {
+      console.error(err.stack);
```

Comment:
doesnt do anything when deployed.
should this go to a log file in the future?

Reply:
Was working on writing my own logger a couple months ago. Haven't had time to finish yet.